packages/agent-toolkit/src/core/tools/monday-apps-tools/consts/routes.consts.ts

```diff
@@ -6,47 +6,34 @@ export enum HttpMethod {
 }
 
 export const MONDAY_APPS_DOMAIN = 'https://monday-apps-ms.monday.com';
-const APPS_MS_PATH = '/apps_ms';
 
 const BASE_API = '/api';
 const BASE_APPS = `${BASE_API}/apps`;
 const BASE_APP_VERSIONS = `${BASE_API}/app-versions`;
 const BASE_CODE = `${BASE_API}/code`;
 const BASE_STORAGE = `${BASE_API}/storage`;
 
-const API_URL = `${MONDAY_APPS_DOMAIN}${BASE_API}`;
 const APPS_URL = `${MONDAY_APPS_DOMAIN}${BASE_APPS}`;
```

Comment:
Is this apps index? We will need to change it later

Reply:
ok